src/base/settingsstorage.h

```diff
@@ -62,4 +62,27 @@ public slots:
     mutable QReadWriteLock m_lock;
 };
 
+template<typename T>
+class SettingsStorageHelper
+{
+public:
+    SettingsStorageHelper(const QString &keyName)
+        : m_keyName(keyName)
+    {
+    }
+
+    void set(const T &value)
+    {
+        SettingsStorage::instance()->storeValue(m_keyName, value);
+    }
+
+    T get(const QVariant &defaultValue = QVariant())
```

Comment:
Thank you for the reply. Excuse me, I should have read the manual before typing.